docs/core/testing/unit-testing-mstest-runner-integrations.md

```diff
@@ -0,0 +1,51 @@
+---
+title: MSTest runner integrations
+description: Learn how to run MSTest runner tests through MSBuild and dotnet test.
+author: nohwnd
+ms.author: jajares
+ms.date: 12/20/2023
+---
+
+# MSTest runner integrations
+
+MSTest runner integrates with MSBuild and `dotnet test` to provide a simple way to run all tests in a solution (`.sln`) or in a single project.
+
+## `dotnet test` integration
```

Comment:
These H2 headers need at least a sentence after them introducing the section. This impacts SEO and prevents the article from looking bad.